### ISSUE TITLE
[FEAT] MAIN: Projects, Latest Drops section Update

### ISSUE THREAD

**Issue by dudlyart** [Redlion-news/issues#11]
![image](https://user-images.githubusercontent.com/81701877/198041211-2012f7a9-c977-4642-a4d2-565540a6caa2.png)

Few updates on this section:

**Height increase on the boxes**
lets make the small boxes 130px height
![image](https://user-images.githubusercontent.com/81701877/198049382-50051360-0734-477b-99d3-8cc599fbf78c.png)

**Re-enable Shows column**
Let's show the show column, but let's rename it to **Don't Miss** and let's make it one banner as per the design below
We can use the Shows content model for it so we can customize the banner + the Studios one as described below.

Last box we title as **Studios Drops** and would be cool to customize that banner also on CMS, so we can show what we like to.
![image](https://user-images.githubusercontent.com/81701877/198050456-155225fa-c665-403c-9f47-13a72436c0f0.png)

**Comment by qruz-hq** [comment]
Yup, I'll make it in the CMS so you can choose between showing the latest drop and a custom one

**Comment by dudlyart** [comment]
Perfect, plus re-enable the shows, but lets rename the label on front end. Great
